tools/js-sdk-release-tools/src/xlc/codeUpdate/updateUserAgent.ts

```diff
@@ -23,6 +24,13 @@ export function updateUserAgent(packageFolderPath: string, packageVersion: strin
             })
             break;
         case SDKType.ModularClient:
+            // Check if it's a model-only package for MLC
+            const isMlcModelOnly = await isModelOnly(packageFolderPath);
+            if (isMlcModelOnly) {
+                logger.info(`MLC package in ${packageFolderPath} is a model-only package, skipping user agent update`);
```

Comment:
```suggestion
                logger.info(`Modular client package in ${packageFolderPath} is a model-only package, skipping user agent update`);
```
Please don't use MLC/HLC/RLC on log, and update in other logs